packages/styles/scss/patterns/tableofcontents/_tableofcontents.scss

```diff
@@ -0,0 +1,116 @@
+//
+// Copyright IBM Corp. 2016, 2018
+//
+// This source code is licensed under the Apache-2.0 license found in the
+// LICENSE file in the root directory of this source tree.
+//
+
+@import '@carbon/grid/scss/grid.scss';
+@import '@carbon/ibmdotcom-styles/scss/components/layout/_layout.scss';
```

Comment:
@sannminwin since you're already in the styles package, you can reference this relatively.